railties/lib/rails/generators/actions.rb

```diff
@@ -288,14 +292,12 @@ def log(*args) # :doc:
         def execute_command(executor, command, options = {}) # :doc:
           log executor, command
           env = options[:env] || ENV["RAILS_ENV"] || "development"
-          rails_env = " RAILS_ENV=#{env}" unless options[:without_rails_env]
```

Comment:
Why are we inlining this?

Reply:
I'm not sure I understand the question.  Are you referring to the concerns you brought up in https://github.com/rails/rails/pull/34980#discussion_r249245219?